src/shared/components/home/signup.tsx

```diff
@@ -144,7 +144,7 @@ export class Signup extends Component<any, State> {
   registerForm() {
     const siteView = this.state.siteRes.site_view;
     return (
-      <form onSubmit={linkEvent(this, this.handleRegisterSubmit)}>
+      <form class="was-validated" onSubmit={linkEvent(this, this.handleRegisterSubmit)}>
```

Comment:
Huh?

Reply:
It causes fields to show a check mark or an red exclamation mark to indicate whether or not the value is valid

https://getbootstrap.com/docs/5.0/forms/validation/